front/src/modules/rollingStock/components/RollingStockCurve.tsx

```diff
@@ -109,7 +109,7 @@ function Legend(props: {
           {isOnEditionMode && !showPowerRestriction && curve.electrical_profile_level}
           {!isOnEditionMode && !showPowerRestriction && curve.mode}
           {/* TODO: remove this condition when getRollingStock endpoint returns comfort 
-                with type Comfort instead of RollingStockComfortType */}
+                with type Comfort instead of Comfort */}
           {curve.comfort !== STANDARD_COMFORT_LEVEL &&
             !isOnEditionMode &&
             comfort2pictogram(curve.comfort === 'AC' ? 'AIR_CONDITIONING' : curve.comfort)}
```

Comment:
You can remove the comment and the condition

```suggestion
          {curve.comfort !== STANDARD_COMFORT_LEVEL &&
            !isOnEditionMode &&
            comfort2pictogram(curve.comfort)}
```